src/components/TodoModal/TodoModal.tsx

```diff
@@ -2,11 +1,39 @@
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+
+type Props = {
+  selectedTodo: Todo;
+  setSelectedTodo: React.Dispatch<React.SetStateAction<Todo | null>>;
+};
+
+export const TodoModal: React.FC<Props> = ({
+  selectedTodo,
+  setSelectedTodo,
+}) => {
+  const [isLoading, setIsLoading] = useState(false);
+  const [user, setUser] = useState<User | null>(null);
+
+  const { id, title, completed, userId } = selectedTodo;
+
+  useEffect(() => {
+    setIsLoading(true);
+    getUser(userId)
+      .then(setUser)
+      .finally(() => setIsLoading(false));
+  }, [userId]);
+
+  const handleCloseModal = () => {
```

Comment:
The `handleCloseModal` function is used as an event handler. It's recommended to follow a naming convention for event handlers. Consider renaming this function to `onCloseModal` to follow the convention.